### DISCUSSION TITLE
公告显示区域变窄，前端显示问题

### DISCUSSION THREAD

**Discussion by gaoran1209** [tangly1024/NotionNext#2327] [Q&A]
最近某次更新后，发现Hexo 主题下的公告栏，显示区域变窄了。
之前是好的。
![image](https://github.com/tangly1024/NotionNext/assets/15261543/4fe0a3c6-4438-42a5-a9b8-227546161760)
看了下其他主题，也都不正常了
<img width="1429" alt="image" src="https://github.com/tangly1024/NotionNext/assets/15261543/ec2f871a-3d26-42f6-997d-68d1df815d6e">
<img width="1409" alt="image" src="https://github.com/tangly1024/NotionNext/assets/15261543/6f6af9a8-e4c0-4c29-9e8e-83cd39914583">

**Comment by snape2019** [comment]
同样问题加一，好不容易更新博客，却发现公告变窄了，经过我的调试，可以自己做临时修改，方法是使用notion config里面的globe css，最后加上 #notion-article > main{padding-left: calc(min(10px, 8vw));padding-right: calc(min(10px, 8vw));}，我是根据我自己博客的内容作出修改的，可以调整括号里面的10px数值看哪个更适合自己。
